src/wasm.rs

```diff
@@ -134,3 +138,78 @@ impl Server {
     pub fn register_tag_keys_callback(&mut self, _f: Function) {}
     pub fn register_tag_values_callback(&mut self, _f: Function) {}
 }
+
+// Flux crate packages migrated over
+/// (Generated by WASM.)
```

Comment:
This function isn't generated by wasm, but maybe we just need a better docstring in general? What does the function do?